sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -274,6 +273,17 @@ def get_additional_columns(row):
         axis='columns',
     )
 
+    def _upd_a10_gpu_count(row):
+        new_gpu_cnt = AZURE_FRACTIONAL_A10_INS_TYPE_TO_NUM_GPUS.get(
+            row['InstanceType'])
+        if new_gpu_cnt is not None:
+            return new_gpu_cnt
+        return row['AcceleratorCount']
+
+    # Manually update the GPU count for fractional A10 instance types.
+    df_ret['AcceleratorCount'] = df_ret.apply(_upd_a10_gpu_count,
+                                              axis='columns')
```

Comment:
Could we say more in the comment for why we need to do it manually?

Reply:
Good point! Added. PTAL